src/Monolog/Utils.php

```diff
@@ -22,4 +22,33 @@ public static function getClass($object): string
 
         return 'c' === $class[0] && 0 === strpos($class, "class@anonymous\0") ? get_parent_class($class).'@anonymous' : $class;
     }
+
+    private static function hasMbString(): bool
+    {
+        static $hasMbString;
+
+        if (null === $hasMbString) {
+            $hasMbString = extension_loaded('mbstring');
+        }
+
+        return $hasMbString;
+    }
+
+    public static function strlen(string $string, ?string $encoding = null): int
+    {
+        if (self::hasMbString()) {
```

Comment:
I think inlining the `extension_loaded('mbstring')` instead of the function call is more efficient here, as opcache can optimize this call away AFAIK.